src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -338,3 +341,12 @@ func (updated *Config) updateFrom(updateBytes []byte) error {
 	}
 	return nil
 }
+
+func inHomeDir(filename string) string {
+	log.Tracef("Determining user's home directory")
+	usr, err := user.Current()
+	if err != nil {
+		panic(fmt.Errorf("Unable to determine user's home directory: %s", err))
```

Comment:
Since there's no conceivable way to recover from not finding the user's home directory, I just changed that to log.Fatalf.

FYI - genconfig is a small utility program and not part of the lantern program itself.